lib/obj.js

```diff
@@ -731,6 +731,7 @@ function streamFromSharks(req, res, next) {
         res.header('Content-Length', md.contentLength);
         res.header('Content-MD5', md.contentMD5);
         res.header('Content-Type', md.contentType);
+        res.header('Accept-Ranges', 'bytes');
```

Comment:
Should we send this in both the contentLength == 0 and the HEAD case or just the HEAD case? I guess muskie technically support a range request in bytes for a zero length object?